pkg/proxy/util/utils.go

```diff
@@ -261,20 +194,6 @@ func AppendPortIfNeeded(addr string, port int32) string {
 	return fmt.Sprintf("[%s]:%d", addr, port)
 }
 
-// ShuffleStrings copies strings from the specified slice into a copy in random
-// order. It returns a new slice.
-func ShuffleStrings(s []string) []string {
```

Comment:
unused after the removal of the userspace proxy in https://github.com/kubernetes/kubernetes/pull/112133